### PR TITLE
Fix/format merchandise order card datetime and order log orderby desc

### PR DESCRIPTION
**Pull request by LoisChen68**

### CODE REVIEW

**Comment by LoisChen68** [comment]
[(1)【我的主頁】商品紀錄，新買的訂單在上+商品紀錄內，購買時間要是24小時制（16:32才對）](https://trello.com/c/60XfS6g3/2274-1%E3%80%90%E6%88%91%E7%9A%84%E4%B8%BB%E9%A0%81%E3%80%91%E5%95%86%E5%93%81%E7%B4%80%E9%8C%84%EF%BC%8C%E6%96%B0%E8%B2%B7%E7%9A%84%E8%A8%82%E5%96%AE%E5%9C%A8%E4%B8%8A%E5%95%86%E5%93%81%E7%B4%80%E9%8C%84%E5%85%A7%EF%BC%8C%E8%B3%BC%E8%B2%B7%E6%99%82%E9%96%93%E8%A6%81%E6%98%AF24%E5%B0%8F%E6%99%82%E5%88%B6%EF%BC%881632%E6%89%8D%E5%B0%8D%EF%BC%89)

File: src/components/merchandise/MerchandiseOrderCollectionBlock.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ const MerchandiseOrderCard: React.VFC<{
           {orderLog.createdAt && (
             <StyledPurchaseDate className="mb-4 d-flex align-items-center">
               <Icon as={CalendarOIcon} className="mr-2" />
-              {`${moment(orderLog.createdAt).format('YYYY-MM-DD hh:mm')} ${formatMessage(messages.purchase)}`}
+              {`${dayjs(orderLog.createdAt).format('YYYY-MM-DD HH:mm:00')} ${formatMessage(messages.purchase)}`}
```
Comment (clothe09986) [inline src/components/merchandise/MerchandiseOrderCollectionBlock.tsx:144]: Why add `:00` in the end?

**Review by clothe09986 — CHANGES_REQUESTED** [review]

File: src/components/merchandise/MerchandiseOrderCollectionBlock.tsx (hunk 1 above)
Comment (LoisChen68) [reply src/components/merchandise/MerchandiseOrderCollectionBlock.tsx:144]: I forgot where I saw the suffix  `:00` so I added it without much thought.

I just went to find where I saw it, but it looks like I misread it at that time.

If it's not necessary, I can remove it.

Comment (clothe09986) [reply src/components/merchandise/MerchandiseOrderCollectionBlock.tsx:144]: I think it's not necessary.

Comment (LoisChen68) [reply src/components/merchandise/MerchandiseOrderCollectionBlock.tsx]: OK.
modified.

**Review by clothe09986 — APPROVED** [review]

**Pull request merged**